x/ibc/core/spec/07_params.md

```diff
@@ -0,0 +1,18 @@
+<!--
+order: 7
+-->
+
+# Parameters
+
+## Clients
+
+The ibc clients contain the following parameters:
+
+| Key              | Type | Default Value |
+|------------------|------|---------------|
+| `AllowedClients`    | []string | `"Solo Machine","Tendermint"`        |
```

Comment:
If we are using them as data the names should maybe be lowercase without spaces, e.g. `06-solo-machine`, `07-tendermint` (I also think including ICS number is a good idea)

Reply:
It needs to match the client type string, that's why I'm using those

Reply:
we could update the client type to those